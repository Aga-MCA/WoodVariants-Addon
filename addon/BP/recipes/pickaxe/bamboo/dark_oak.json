{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "bamboo_dark_oak_pickaxe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PPP",
      " S ",
      " S "
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:dark_oak_stick"
      },
      "P": {
        "item": "aga_wood_variants:bamboo_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:bamboo_dark_oak_pickaxe"
    }
  }
}