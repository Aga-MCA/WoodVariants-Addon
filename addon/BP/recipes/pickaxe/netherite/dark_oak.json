{
  "format_version": "1.17",
  "minecraft:recipe_smithing_transform": {
    "description": {
      "identifier": "netherite_dark_oak_pickaxe"
    },
    "tags": [
      "smithing_table"
    ],
    "template": "minecraft:netherite_upgrade_smithing_template",
    "base": "aga_wood_variants:diamond_dark_oak_pickaxe",
    "addition": "minecraft:netherite_ingot",
    "result": "aga_wood_variants:netherite_dark_oak_pickaxe"
  }
}