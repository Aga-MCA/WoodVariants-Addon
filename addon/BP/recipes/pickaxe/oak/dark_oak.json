{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "oak_dark_oak_pickaxe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PPP",
      " S ",
      " S "
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:dark_oak_stick"
      },
      "P": {
        "item": "aga_wood_variants:oak_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:oak_dark_oak_pickaxe"
    }
  }
}