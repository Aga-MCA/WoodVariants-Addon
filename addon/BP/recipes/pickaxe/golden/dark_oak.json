{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_dark_oak_pickaxe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PPP",
      " S ",
      " S "
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:dark_oak_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_dark_oak_pickaxe"
    }
  }
}